### DISCUSSION TITLE
Rename examples with postfix for easier comparison

### DISCUSSION THREAD

**Discussion by peter-scholtens** [moka-rs/moka#360] [Ideas]
In the examples directory, when **async** and **sync** are used as [postfix](https://en.wikipedia.org/wiki/Suffix) instead of prefix, the user may easily compare similar implementations of the examples for synchronous and asynchronous implementation of the cache. And would it be handy to use **sync** and **future** for better visual distinction?
![Screenshot 2023-12-21 at 11-03-15 moka_examples at main · moka-rs_moka](https://github.com/moka-rs/moka/assets/7198614/2bbb9305-6d63-40ad-b486-d00dddfacb31)

**Comment by tatsuya6502** [comment]
Thank you for the suggestion. Yes, I agree that it will be much clearer to use async (`_future`) and `_sync` as suffixes than the other.

**Reply by tatsuya6502** [reply]
I am going to work on this today.

**Reply by tatsuya6502** [reply]
Created a PR:

- #371

**Reply by tatsuya6502** [reply]
Done. Merged it into the `main` branch.
